ext/pdo_firebird/firebird_driver.c

```diff
@@ -599,6 +599,21 @@ static int pdo_firebird_fetch_error_func(pdo_dbh_t *dbh, pdo_stmt_t *stmt, zval
 }
 /* }}} */
 
+/* Check if connection is liveness */
+static int pdo_firebird_check_liveness(pdo_dbh_t *dbh){
+	ISC_STATUS status[20];
```

Comment:
is this 20 here have a more meaningful alternative?

Reply:
According to the documentation, I need to use 20 elements vector

http://docwiki.embarcadero.com/InterBase/2020/en/Setting_Up_an_Error_Status_Vector
http://docwiki.embarcadero.com/InterBase/2020/en/Isc_database_info()

and all the sample codes I consulted also use 20 elements, so I think that using a different sized vector can crash the application

Reply:
however, using NULL instead of the 20 position array worked, but I think it’s not reliable